pennylane/ops/qubit.py

```diff
@@ -1502,6 +1580,9 @@ def decomposition(phi, lam, wires):
         ]
         return decomp_ops
 
+    def adjoint(self, do_queue=False):
+        raise NotImplementedError
```

Comment:
Oh, if this raises not implemented, I assume that it will be decomposed and _then_ inverted? Clever!

Reply:
That would be clever! But that's not what I intended lol.

Reply:
Maybe it Just Works™?